recipes/cx_logging/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "cx_Logging" %}
+{% set version = "3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ba8a7465facf7b98d8f494030fb481a2e8aeee29dc191e10383bb54ed42bdb34
+
+build:
+  number: 0
+  skip: True  # [not win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - cx_Logging
+
+about:
+  home: https://github.com/anthony-tuininga/cx_Logging
+  license: PSF-2.0
```

Comment:
The [license in the repo](https://github.com/anthony-tuininga/cx_Logging/blob/main/LICENSE.txt) seems to be BSD-3-clause, not PSF?

Reply:
I copied from cx_Freeze assuming it would be the same license. Fixed.